docker/resources/build_fuzzy_libmath_dockerfile.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+DOCKERFILE=Dockerfile.mcalibmath
+
+generate_docker() {
+    
+cat > ${DOCKERFILE} <<HERE
+# Base image
+FROM ${1}
+
+# Copy fuzzy environment from fuzzy image
+RUN mkdir -p /opt/mca-libmath
+COPY --from=verificarlo/fuzzy:v0.4.1-lapack /opt/mca-libmath/libmath.so /opt/mca-libmath/
+COPY --from=verificarlo/fuzzy:v0.4.1-lapack /usr/local/lib/libinterflop* /usr/local/lib/
```

Comment:
do we want to pass the fuzzy tag as an optional parameter defaulting to what it is now?